settings.gradle

```diff
@@ -19,6 +19,7 @@ include 'degrader'
 include 'entity-stream'
 include 'li-jersey-uri'
 include 'li-protobuf'
+include 'pegasus-shaded-netty'
```

Comment:
I suggest renaming this subproject to `li-netty` for consistency (since this is close to what is done for `li-jersey-uri` and `li-protobuf` though without the shading)

Reply:
+1

Reply:
Good idea. I've renamed it to li-netty.